src/Bicep.Core.IntegrationTests/ExtensibilityTests.cs

```diff
@@ -339,6 +339,26 @@ public void Kubernetes_import_existing_connectionstring_test()
             result.Should().NotHaveAnyDiagnostics();
         }
 
+        [TestMethod]
+        public void Kubernetes_CustomResourceType_EmitWarning()
+        {
+            var result = CompilationHelper.Compile(Services, """
+                import 'kubernetes@1.0.0' with {
+                  namespace: 'default'
+                  kubeConfig: ''
+                }
+                resource crd 'custom/Foo@v1' = {
```

Comment:
How do we distinguish whether the user intended to define a Kubernetes fallback type vs an ARM fallback type?

Reply:
In that case users have to fully qualify the resource type. Added a test for that: https://github.com/Azure/bicep/pull/10598/files#diff-d3b11a4daaf2569ef3d245e9d62d1745ebf0cadc68056635554f3c19ca911f69R364